spark/sql-13/src/main/scala/org/elasticsearch/spark/sql/DefaultSource.scala

```diff
@@ -533,10 +534,10 @@ private[sql] case class ElasticsearchRelation(parameters: Map[String, String], @
 
       // perform a scan-scroll delete
       val cfgCopy = cfg.copy()
+      InitializationUtils.setUserProviderIfNotSet(cfgCopy, classOf[HadoopUserProvider], null)
```

Comment:
Should we just roll `setUserProviderIfNotSet` into `discoverClusterInfo` to keep us from accidentally calling `discoverClusterInfo` without it in the future?

Reply:
I don't disagree, but I think the problems are rooted deeper than just this initialization call. Honestly, I think all of `InitializationUtils` in general needs to be put on a bus.